### PR TITLE
[jsk_robot_startup/util] Support python3 in mux_selector.py

### PR DESCRIPTION
**Pull request by tkmtnt7000**
In python3, we have to use `_thread` instead of `thread`.

### CODE REVIEW

File: jsk_robot_common/jsk_robot_startup/util/mux_selector.py [hunk 1]
```diff
@@ -19,13 +19,15 @@
...
 import sys
 import roslib.message
 import rostopic
 from topic_tools.srv import MuxSelect
 import traceback
-
+if sys.version.major == 2:
+    import thread
+else:
+    import _thread
```
Comment (iory) [inline jsk_robot_common/jsk_robot_startup/util/mux_selector.py:30]: How about writing `import _thread` as `import _thread as thread`?

**Review by iory — CHANGES_REQUESTED** [review]

File: jsk_robot_common/jsk_robot_startup/util/mux_selector.py (hunk 1 above)
Comment (tkmtnt7000) [reply jsk_robot_common/jsk_robot_startup/util/mux_selector.py]: Thank you for advice. I think that's a good idea.
Modified in [`642da39` (#1517)](https://github.com/jsk-ros-pkg/jsk_robot/pull/1517/commits/642da39f4a9f6e8298ced5e6cbbcfca0d9c7825f)

Comment (tkmtnt7000) [reply jsk_robot_common/jsk_robot_startup/util/mux_selector.py]: sorry, I made a mistake.

Comment (iory) [reply jsk_robot_common/jsk_robot_startup/util/mux_selector.py]: Following changes will be good.
```
import sys
if sys.version_info.major < 3:
    import thread
else:
    import _thread as thread
```

Comment (tkmtnt7000) [reply jsk_robot_common/jsk_robot_startup/util/mux_selector.py]: Thanks. I fixed.

**Review by iory — APPROVED** [review]

**Pull request merged**